components/benefit_sponsors/app/views/benefit_sponsors/profiles/registrations/_sic_help.html.erb

```diff
@@ -45,6 +42,7 @@
       $("#sicHelperWrapper").toggleClass('hidden');
       $('#<%=sicFieldId%>').val(data.sic_code);
       $('#<%=sicFieldId%>').trigger('chosen:updated');
+			document.getElementById('agency_organization_profile_attributes_sic_code').value = data.sic_code;
```

Comment:
Sic-Code value is updated to the Field on selecting the sic code from the helper. Changes look good.